File: src/NBdbm/Classes/Interfaces/iCX.vb
Namespace Interfaces

  Namespace iCX

    Public Interface iCentroCusto
      Inherits Primitivas.iObjetoTabela
      Property centroCusto() As String
    End Interface

    Public Interface iCaixa
      Inherits Primitivas.iObjetoTabela
      Property idCentroCusto() As Integer
      Property idEntidade() As Integer
      Property pagarOUreceber() As Boolean
      Property descricaoConta() As String
      Property vlrConta() As Integer
      Property dtVencimento() As Date
      Property dtPagamento() As Date
      Property acrescimos() As Integer
      Property abatimento() As Integer
      Property vlrPago() As Integer
    End Interface

  End Namespace
End Namespace

File: src/NBdbm/CS/Classes/Interfaces/iCX.cs
using System.Data;
using System.Collections;
using Microsoft.VisualBasic;
using System.Diagnostics;
using System;

namespace NBdbm.Interfaces.iCX
{
    public interface iCentroCusto : Primitivas.iObjetoTabela
    {
        string centroCusto
        {
            get;
            set;
        }
    }

    public interface iCaixa : Primitivas.iObjetoTabela
    {
        int idCentroCusto
        {
            get;
            set;
        }
        int idEntidade
        {
            get;
            set;
        }
        bool pagarOUreceber
        {
            get;
            set;
        }
        string descricaoConta
        {
            get;
            set;
        }
        int vlrConta
        {
            get;
            set;
        }
        DateTime dtVencimento
        {
            get;
            set;
        }
        DateTime dtPagamento
        {
            get;
            set;
        }
        int acrescimos
        {
            get;
            set;
        }
        int abatimento
        {
            get;
            set;
        }
        int vlrPago
        {
            get;
            set;
        }
    }
}
